packages/helper-image/test/unit/spec/image-helper.browser.js

```diff
@@ -0,0 +1,137 @@
+/**!
```

Comment:
The file name won't control node vs browser tests. ideally, your tests should be structured such that they don't care what environment they run in, but if you absolutely need different browser and node tests, there are helpers in test-helper-mocha for that.